Lab4.ipynb

```diff
@@ -458,46 +583,474 @@
Xrk2bue1hiZosn8mKzNa4XPTdMag50xrE/MyrXt43077KfQivvfWvH5O9i+7RERERERERERkMfixS0REREREREREFoMfu0REREREREREZDH4sUtERERERERERBaDH7tERERERERERGQxHGob43z9es3DnbX7o62scoV/NGmQiYesB8FENIG1ZwRrEdnXWmYysHdgGzRWoR20cpERKpiPqiqbOshySXY4MlXQ2AbLBM0lzjFZZFJfyIAyNZ+TrBnB2oQWs00wXea74/NMF3dyG/CbE5h1xtOn8m8G8wraPsg0RiYRMsYExq1sWUO7WcOYiHY8omOwA7MQSGfYrJoMOMmAUsVWv0tgtCS7Tog3nHvKEx0DWdM4SbZMnM/UftP8is+Z4gqawDw2gLWIYiLNMxiEaN8kw1PKk2glAwMirk20s66u2e45AG1TYZ/FvAy5HU3Qaa+GNUX7DK57WBMpj3X3RxzD9PxkU6NcWI22CZofigk4M027YU+Ge9FcSUY6HJeGmQtiZbpwMV6PezX1g/YCyge0pwTIfkk5tZWX557NlM6046mwX8F66Fiwq9gQN4b2p0uXchtgXcR1n+x4FJt05oZ3Djw3BHDvJXM9nS/TmNOzkxke7cbwXpRsofSuRIZXeOfKpkcyCsNzkoUVYijFM75v4fkSTMNhLbf39b0cK+PWauyT9Xb//IXcBphIaS2PZ7dXL9K+TnvVJsRKOmM01lRVse37OuT3hiEb36spH6S4oneFdC/YORP+ZZeIiIiIiIiIiCwGP3aJiIiIiIiIiMhi8GOXiIiIiIiIiIgsBj92iYiIiIiIiIjIYvBjl4iIiIiIiIiILIZDbWOscVy1GZDVpFZtCMk6WFU1717JbZCRMNnkyN4HDNtbuS9ko+mY08Aah1bD9JxkoLoBBgcYq2jCoLHC3wRjV6MdNEyQSYQMSmF+os2mim0sZJchmxO1HxuHuScLyIlVM8pAYwKGEbIukqkj3nslG1PQvJfsN9Q23Dt0LF51QLyFdjBmG3avqqpxO9hlYKzIljMcAXNPupfGG+xMwykwIpHNKcXnTZhLGCua+2g3A9MLGflGMN2QmWs4tWruIcMTrR+y5kX7GpiF0MBF+aBxndYPMe+AJSzFFu1VZPMhczKYvFL7aJUaycAL8xPaQftadx4q9IXinkyCZG0ik1faI2GvQsvpCOeAXTjXJMg6SfmAbFPJ4AdmUTqTcE4N40L9A6LFqwpzZ1qHaI2DOU77SVWxPSw2AnNP/SbLdPpNsC6i7ZsMkA1zJcXbxkMvyH25svq+gPsDnI2m5y7E6yPNZ1ib43335Xspxvdg3acYpxxBexjYsQc6NyQDM+ybaBCNVw+wOwfoLDWRHXAPjHz3n1ttG3IerfsJ9s10lhzBVoyWcoLyXieXNd99k3lxCmuqip+T3jfTGA5gVxzp/E97MhlXU99hHuisR+fOuIdT/8i4Se8LdD5I+Z3OGJSvyKiczjA0ruE353n9fcq/7BIRERERERERkcXgxy4REREREREREVkMfuwSEREREREREZHF4McuERERERERERFZDIe7QH1ghkK2qag3lNWsokL0VHwtFJ/Dws5U6JGKEkMx8lTUr1scjkiFnfF55kax0ape8UIqNoqFqqGQeipO2i2wR4WGO8Xiae5JLEBF9ENfsNDh8VxQd7oCRaNTMULqNxV0pILCVKg7FSmkZ6eCqDSfSR7RlBOkNqqKCxCnIs5UoBIKPo+noKhqKkpMBbZJwNEQPGAB9BO5f1QYnHNTKrwNYgoq+gsFR+Os0RxDGwjlmhRbVJy/Iwih34QCzm0xBV0PuYbmEsUPFJ9pbeY7W7nwVtMgImgU7KWCwihhSIWgqeA+FqyF+0PsU5zgeFPBWpLgpLZp/4ZYnq+B7Cf1keaYCp2T2IWKLKf2KR+cgML6VBg9QcX8AZQZNPbTT77mj8R7H33Hx/KPQvHyGIcNcUZV8bmhIHemtikmKKVQbkrrh9qmAsmwF8Z5pjMdvJ+Q1IfyWzrv4PPQOWWA+Ql9RGkKSZRQJAR7eyqaTUW66ewB55o4VlS0nmQLNyDXwPk/Fkan/EtnJpKthXWP770wP5198FZDq/E8kDyC5FKNdz8cq+ZzpvmJBeQPaAPPNRSfJ8NzgtSG1jfKzFIswzsePiecX/C8k56Tzot0zkcpWIhD3GdCDML3h4R/2SUiIiIiIiIiIovBj10iIiIiIiIiIrIY/NglIiIiIiIiIiKLwY9dIiIiIiIiIiKyGPzYJSIiIiIiIiIii2FxNka0oIzBkEC2HJBGkAki2hjBiDTtXo7XR7KAkFkomAnQTgTWpmRdrMqmkulitveRNa5l5iIDFUEmNCKZyfbAgnEsmyrIDjJdWp1PtEWSjYTMlWRMCXaMgaxKwdpZdYC5JxmUupYssOKgZSOYiMgMMt53Nv9msN9UVc/+STaWXbCaNNqJlssqNlbR9fQ4dC/YYmjdJ3vLeHYb7gVLY9f8GnIqWm5oXZHtMEFtgC0HDZCdvlAMNveZZLgaNprWItpPTmUbZVzLaMGDuSdT3/Uw92TxImsRmQfJbJfaB7PQeO6+eH26uJPbDnmS1hrmdpqfZGsjoyPkK2y7aaGKQH7r2Io7VsiqAwyqtJaDTY/ifoK+oCUs7G1oKaSxOgKxT4QYf/QffzTf27SwDsfDmZEMr3R+gbWMtrZknaTfJOh5UozDvdMlOOtSvKX9Pt95wDtHz8gd26czBhkQKY83zPV4zu+cL6ty35tzT2s5/ibtJ53z4gH3570AzsVk7KV32esh19Bao32Q9k2Yn7Qmxv3GmaEKzdHRmP7lD+e2P/WZfJ3OaSEm0HZNa43SMr0/BzviDO9nXWtpfO9v2iWx33DujuMCz1M34N2c2g5rHGOWxmpN/MsuERERERERERFZDH7sEhERERERERGRxeDHLhERERERERERWQx+7BIRERERERERkcXgxy4REREREREREVkMy7Mxkqkv2T7I
```

Comment:
It could be better for us to visualize the pattern if the plot is not in scatter and has more data included.